sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayoutController.java

```diff
@@ -419,7 +419,11 @@ public boolean onKeyDown(int keyCode, KeyEvent event) {
         sendEvent(CONTROLLER_KEY_PRESS_EVENT, null);
         break;
       case KeyEvent.KEYCODE_MEDIA_REWIND:
-        _player.seek(_player.getPlayheadTime() - REWIND_STEP); // << -10sec
+        int timeAfterSeek = _player.getPlayheadTime() - REWIND_STEP;
+        if(timeAfterSeek < 0) {
+          timeAfterSeek = 0;
+        }
```

Comment:
Use smth like this: `= Math.max(0, ..)` instead of `if(timeAfterSeek < 0) { ...`